readline-complete.el

```diff
@@ -135,10 +135,18 @@
 ;; editline instead of readline. Consider recompiling bash, python,
 ;; etc with readline support instead.
 
+;; *** Note: emacs contained a bug from ~24.3.91 to ~24.5-rc3-fixed where
+;; completion on longer lines would fail and possibly erase your prompt.
+
 ;; Patches welcome! https://github.com/monsanto/readline-complete.el
 
 ;;; Code:
 
+(when (or (and (= 24 emacs-major-version) (>= 3 emacs-minor-version))
+          (and (= 25 emacs-major-version) (<= 0 emacs-minor-version)))
```

Comment:
Unless this is going in `25.1`, you may want to extract the exact patch version from `emacs-version` instead of using `emacs-minor-version`. Otherwise someone updating to the latest may be confused that they still get the warning.

For reference, here is how `emacs-*-version` are defined:

```elisp
(defconst emacs-major-version
  (progn (string-match "^[0-9]+" emacs-version)
         (string-to-number (match-string 0 emacs-version)))
  "Major version number of this version of Emacs.
This variable first existed in version 19.23.")

(defconst emacs-minor-version
  (progn (string-match "^[0-9]+\\.\\([0-9]+\\)" emacs-version)
         (string-to-number (match-string 1 emacs-version)))
  "Minor version number of this version of Emacs.
This variable first existed in version 19.23.")
```